README.md

```diff
@@ -4,13 +4,13 @@
 
 ## Introduction
 
-Crucible takes [multiple performance tooling projects](config/default_subprojects) and integrates them, with the intention to provide a well rounded, portable, and highly functional end-to-end performance tool harness. If successful, crucible can be used for test automation which can execute, measure, store, visualize, and analyze the performance of various systems-under-test.
+Crucible is a performance tool harness that integrates [multiple performance tooling projects](config/default_subprojects). The project's intention is to provide a well rounded, portable, and highly functional end-to-end performance testing harness. Crucible can be used for test automation to execute, measure, store, visualize, and analyze the performance of various systems-under-test (SUT).
 
 ## Features
 
 ### Zero-touch Installation
 
-Crucible should only need to be installed on a single Linux system (the crucible-controller), and the user should not have to install crucible on any target host, cluster, or cloud that they wish to test. Crucible uses container images to satisfy nearly all software dependencies. The Linux system where you installed crucible-controller only needs to have [podman](https://podman.io) software installed. Endpoints (systems, clusters, clouds that are "under-test") only need the ability to pull container images. Most clusters/clouds have this ability already, and non-clouds (like a remote-host) only need podman.
+Crucible can be installed on a single Linux system (the crucible-controller), and does not require installation on any target host, cluster, or cloud that needs  testing. Crucible uses container images to satisfy software dependencies. The crucible-controller system only needs to have [podman](https://podman.io) software installed. Endpoints (systems, clusters, clouds that are "under-test") only need the ability to pull container images. Most clusters/clouds have this ability already, and non-clouds (like a remote-host) only need podman.
```

Comment:
We should add `git` as a requirement along with podman.

Reply:
Updated in new commit.